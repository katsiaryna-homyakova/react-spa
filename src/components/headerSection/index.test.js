import React from "react";
import Header from "./index";
import { shallow } from "enzyme";

describe("Header Section component", () => {
  it("renders correctly", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
