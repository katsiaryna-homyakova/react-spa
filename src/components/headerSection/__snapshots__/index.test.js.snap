// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Section component renders correctly 1`] = `ShallowWrapper {}`;
